src/modules/75-cf/components/FlagActivation/ServicesList.tsx

```diff
@@ -78,6 +83,7 @@ const EditServicesModal: FC<EditServicesProps> = ({
             intent="primary"
             variation={ButtonVariation.PRIMARY}
             disabled={loading}
+            onClick={() => onSave()}
```

Comment:
Nit: when you're only invoking a function and not passing anything to it, you can pass a reference:
```suggestion
            onClick={onSave}
```

Reply:
done